vm/src/pyobject.rs

```diff
@@ -693,6 +713,12 @@ impl<T: ?Sized + PyObjectPayload> IdProtocol for PyObject<T> {
     }
 }
 
+impl IdProtocol for PyObjectRc {
```

Comment:
Same here if this was in pyobjectrc.rs then it could simply reference the inner value.